src/bakery_manager_mobile/lib/pages/my_account.dart

```diff
@@ -106,229 +160,155 @@ class MyAccountPageState extends State<MyAccountPage> {
+                              ),
+                            ],
                           );
                         }).toList(),
                       ),
-                    ],
-                  ),
-                ),
-              ),
 
-              // Buttons at the bottom
-              Padding(
-                padding: const EdgeInsets.symmetric(
-                    horizontal: 16.0, vertical: 16.0),
-                child: Column(
-                  children: [
-                    ElevatedButton.icon(
-                      style: ElevatedButton.styleFrom(
-                        backgroundColor: Colors.orange,
-                        padding: const EdgeInsets.symmetric(
-                            vertical: 16, horizontal: 32),
-                        shape: RoundedRectangleBorder(
-                          borderRadius: BorderRadius.circular(10),
-                        ),
-                      ),
-                      onPressed: () {
-                        Navigator.pushNamed(context, editAccountPageRoute);
-                      },
-                      icon: const Icon(Icons.edit),
-                      label: const Text('Edit Account'),
-                    ),
-                    const SizedBox(height: 16),
-                    ElevatedButton(
-                      onPressed: () {
-                        Navigator.pushReplacementNamed(context, '/login');
-                      },
-                      style: ElevatedButton.styleFrom(
-                        backgroundColor: Colors.red,
-                        padding: const EdgeInsets.symmetric(
-                            vertical: 16, horizontal: 32),
-                        shape: RoundedRectangleBorder(
-                          borderRadius: BorderRadius.circular(10),
+                      const SizedBox(height: 64),
+
+                      // Action Buttons
+                      Center(
+                        child: Column(
+                          children: [
+                            ElevatedButton.icon(
+                              style: ElevatedButton.styleFrom(
+                                backgroundColor: const Color.fromARGB(255, 209, 125, 51),
+                                padding: const EdgeInsets.symmetric(
+                                    vertical: 16, horizontal: 32),
+                                shape: RoundedRectangleBorder(
+                                  borderRadius: BorderRadius.circular(10),
+                                ),
+                              ),
+                              onPressed: () {
+                                Navigator.pushNamed(context, editAccountPageRoute);
+                              },
+                              icon: const Icon(Icons.edit, color: Colors.white),
+                              label: const Text(
+                                'Edit Account',
+                                style: TextStyle(color: Colors.white),
```

Comment:
Not sure if edit account is the best wording for this now. Maybe change to Update contact information or something similar.